### DISCUSSION TITLE
Any public statement on the futre of this project?

### DISCUSSION THREAD

**Discussion by fad1** [Xyrio/RSSOwlnix#145] [General]
I really liked RSSOwl, any idea if this project will resume ? :)

**Comment by Xyrio** [comment]
i am working on it from time to time (sometimes a year or more can pass). currently stuck on a bug. so once i can fix it ill release.

**Comment by Neustradamus** [comment]
No changes since a long time, the latest version is 2.9.0-beta (2021-04-18):
- https://github.com/Xyrio/RSSOwlnix/releases

**Reply by Albirew** [reply]
that's only the releases.
code is still updated albeit not released as a new version: https://github.com/Xyrio/RSSOwlnix/commits/master/
